habitat-lab/habitat/config/default_structured_configs.py

```diff
@@ -198,13 +207,70 @@ class TeleportActionConfig(ActionConfig):
 @attr.s(auto_attribs=True, slots=True)
 class VelocityControlActionConfig(ActionConfig):
     type: str = "VelocityAction"
-    # meters/sec
-    lin_vel_range: List[float] = [0.0, 0.25]
-    # deg/sec
-    ang_vel_range: List[float] = [-10.0, 10.0]
+    lin_vel_range: List[float] = [0.0, 0.3]  # meters/sec
+    ang_vel_range: List[float] = [-0.45, 0.45]  # rad/sec
+    time_step: float = 0.1  # seconds
+    enable_scale_convert: bool = False
+
+
+@attr.s(auto_attribs=True, slots=True)
+class WaypointControlActionConfig(VelocityControlActionConfig):
+    r"""
+    In Navigation tasks only, this action will utilize a velocity controller to
+    move the robot to the desired waypoint.
+    The waypoint is specified in the frame of the robot.
+    """
+    type: str = "WaypointAction"
+    action_duration: float = 1.0  # seconds
+    # Action space range
+    waypoint_lin_range: List[float] = [-0.5, 0.5]  # meters
+    waypoint_ang_range: List[float] = [-np.pi, np.pi]  # radians
+    wait_duration_range: List[float] = [0.0, 10.0]  # seconds
```

Comment:
Should the wait_duration_range be [time_step, 10.] so that the controller atleast takes one step in the env?

Reply:
Good call, although having dependencies between configuration fields seems to be an unnecessary complexity. I'll modify the logic of the loop in `WaypointAction` to always execute for 1 step.